src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,193 @@
+.pf-c-inline-edit {
+  // Group
+  --pf-c-inline-edit__group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Value group
+  --pf-c-inline-edit__group--m-value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__group--m-input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action group
+  --pf-c-inline-edit__action-group--MarginTop: var(--pf-global--spacer--xl);
```

Comment:
why is this `--pf-c-inline-edit__action-group--MarginTop` instead of using `m-action-group`

Reply:
good catch, it's incorrect